frontend/Gruntfile.js

```diff
@@ -72,6 +79,6 @@ module.exports = function(grunt) {
 
   });
 
-  grunt.registerTask('default', ['sass', 'cssmin', 'uglify', 'copy']);
+  grunt.registerTask('default', ['sass', 'cssmin', 'copy']);
```

Comment:
uglify was actually helpfull, if you need to see the code for debug consider having two task:
```
grunt.registerTask('build_development', ['sass', 'cssmin', 'copy']);
grunt.registerTask('build_production', ['sass', 'cssmin', 'uglify', 'copy']);
```
maybe with different names.